Makefile

```diff
@@ -234,6 +234,13 @@ test/k8s-%: test/k8s/clean test/k8s/cluster-%
 	sleep 5s
 	kubectl --context=kind-$(KIND_CLUSTER_NAME) wait deployment wasi-demo --for condition=Available=True --timeout=5s
 
+.PHONY: test/k8s-oci-%
+test/k8s-oci-%: test/k8s/clean test/k8s/cluster-oci-%
```

Comment:
Could we add this to the CI workflow?

Reply:
@jprendes Thank you for the question!
I think this is added to the CI workflow already.
We can see this is executed in the following step of the workflow.
https://github.com/containerd/runwasi/actions/runs/7442612789/job/20246359284?pr=434#step:7:1

Reply:
I think the make target above should cover this `test/k8s-%: test/k8s/clean test/k8s/cluster-%`.

Reply:
Oh, I see, before CI was triggering `test/k8s-%` and now it's triggering `test/k8s-oci-%`.

Thanks!